pkgs/by-name/ga/garmin-grafana/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  fetchFromGitHub,
+  python3Packages,
+  lib,
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "garmin-grafana";
+  version = "0.3.0";
+
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "arpanghosh8453";
+    repo = "garmin-grafana";
+    rev = "v${version}";
+    hash = "sha256-nuVT6LK9KIs/FwUbdfI4xpKru4jfAyj1/vmk7ji43zk=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = with python3Packages; [
+    fitparse
+    garminconnect
+    influxdb
+    influxdb3-python
+    python-dotenv
+  ];
+
+  doCheck = false; # there are no tests
+
+  dontCheckRuntimeDeps = true;
+
+  passthru.grafana-dashboard = "${src}/Grafana_Dashboard/Garmin-Grafana-Dashboard.json"; # can be provisioned to Grafana
```

Comment:
Do we need to pull in the whole src closure here? The JSON file could be copied into the package's `$out` as a `postInstall` step.